applications/teletype/src/templates/teletype_account_zone_change.erl

```diff
@@ -82,3 +88,12 @@ process_req(DataJObj) ->
         'ok' -> teletype_util:send_update(DataJObj, <<"completed">>);
         {'error', Reason} -> teletype_util:send_update(DataJObj, <<"failed">>, Reason)
     end.
+
+-spec zones_data(kz_json:object()) -> api_object().
```

Comment:
Spec is wrong. Return a JSON object in one clause and a proplist of one tuple in the other.